correct.py

```diff
@@ -153,7 +153,7 @@ def encode_sentence(sent: Sentence) -> Dict[str, Any]:
         nonlocal offset
         for ix, t in enumerate(sent.tokens):
             tokens[ix]["i"] = offset
-            offset += len(t.original or "")
+            offset += len(t.origin_spans or "")
```

Comment:
Why is this better than `len(t.original or "")`?

Reply:
This has been reverted and other code updated to fit that.